bin/testEnvRootMinikube.sh

```diff
@@ -2,6 +2,11 @@
 
 export K8S_VER=${K8S_VER:-v1.9.2}
 
+if [ "$ARCH" == "ppc64le" ]; then
+    echo "Minikube tests not supported on ppc64le, exiting..."
+    exit 1
+fi
+
 if [ ! -f /usr/local/bin/minikube ]; then
```

Comment:
Don't you want to also cross-compile (as we do for mac and arm ? )

Reply:
I am assuming you are referring to this (`.circrleci/config.yml`):
```            make build GOOS=darwin KUBECONFIG=/go/src/istio.io/istio/.circleci/config
            make build GOOS=linux GOARCH=arm KUBECONFIG=/go/src/istio.io/istio/.circleci/config
```
The problem with this is, circleci doesn't support ppc64le hardware yet. We would definitely want to test istio with kubernetes on ppc64le. Is there any alternative approach to do that?

Reply:
@costinm never mind. I just realised its just cross-compilation anyway :) , yes, we can add that for ppc64le as well. I will remove this change and try out cross-compiling for ppc64le as well.